### PR TITLE
Membership UI improvements

### LINKED ISSUE #612: Membership UI improvements
- [x]  Use single term - Revoke membership flow currently has 3 terms used. Revoke, Delete and Remove. Need to stick to one. 

- [x]  Show notification after action is completed - When a member has been added or removed, show notification that this action has been succesfully completed (same UI as bookmark currently uses)

- [x]  Crew Tab - Bottom purple bar isn't showing

- [x]  Accordion animation is very jumpy, this doesn't happen in the accordion in the proposal section...

- [x] Remove sidebar with "how to add member info". Needs to be inside create/edit membership form instead. 

- [x]  UI spec for name tab -https://imgee.s3.amazonaws.com/imgee/7f44123c7cc34ac68f4881e51b962c9b.png

- [x]  UI spec for empty screen (no members) - https://imgee.s3.amazonaws.com/imgee/2ff8e8afff574065b7e1c4bb4ff861c3.png

- [ ]  Alphabetize name list for both Name & Role tabs

### PR DESCRIPTION
**Pull request by vidya-ram**
Fixes #612 except Alphabetize name list for both Name & Role tabs

### CODE REVIEW

File: funnel/views/membership.py [hunk 1]
```diff
@@ -189,6 +189,7 @@ def edit(self):
                 db.session.commit()
                 return {
                     'status': 'ok',
+                    'message': _("The member's roles has been updated"),
```
Comment (iambibhas) [inline funnel/views/membership.py:192]: `have been`

File: funnel/forms/membership.py [hunk 2]
```diff
@@ -41,9 +41,9 @@ class ProjectCrewMembershipForm(forms.Form):
         validators=[forms.validators.DataRequired(_(u"Please select a user"))],
         description=__("Find a user by their name or email address"),
     )
-    is_editor = forms.BooleanField(__("Editor"), default=False)
-    is_concierge = forms.BooleanField(__("Concierge"), default=False)
-    is_usher = forms.BooleanField(__("Usher"), default=False)
+    is_editor = forms.BooleanField(__("Editor"), default=False, description=__("Can edit project details, proposal guidelines, schedule, label and venue "))
```
Comment (jace) [inline funnel/forms/membership.py:44]: "Can edit project details, proposal guidelines, schedule, labels and venues"

File: funnel/forms/membership.py [hunk 3]
```diff
@@ -41,9 +41,9 @@ class ProjectCrewMembershipForm(forms.Form):
...
+    is_concierge = forms.BooleanField(__("Concierge"), default=False, description=__("Can setup events for check-in, add ticket clients and attendees"))
```
Comment (jace) [inline funnel/forms/membership.py:45]: "Can manage participants and see contact info"

File: funnel/forms/membership.py [hunk 4]
```diff
@@ -41,9 +41,9 @@ class ProjectCrewMembershipForm(forms.Form):
...
+    is_usher = forms.BooleanField(__("Usher"), default=False, description=__("Can check-in attendees"))
```
Comment (jace) [inline funnel/forms/membership.py:46]: "Can check-in a participant using their badge at a physical event"

File: funnel/templates/organization_membership.html.jinja2 [hunk 5]
```diff
@@ -99,7 +76,7 @@
                 <button class="mui-btn mui-btn--nostyle mui--text-light" @click="closeForm($event)">Cancel</button>
               </div>
               <div v-if="activeMember" class="modal-form__action-box--revoke mui--text-center">
-                <button @click="fetchForm($event, deleteURL)" title="Delete" class="mui-btn mui-btn--nostyle mui--text-subhead" data-cy-btn="revoke"><span class="mui--text-light">Revoke this membership - </span><span class="mui--text-bold">Revoke</span></button>
+                <button @click="fetchForm($event, deleteURL)" class="mui-btn mui-btn--nostyle mui--text-subhead" data-cy-btn="revoke"><span class="mui--text-light">Remove this member - </span><span class="mui--text-bold">Remove</span></button>
```
Comment (jace) [inline funnel/templates/organization_membership.html.jinja2:79]: "Remove this member" should be clickable. The "- Remove" part is not needed.

File: funnel/templates/project_membership.html.jinja2 [hunk 6]
```diff
@@ -139,7 +94,7 @@
                 <button class="mui-btn mui-btn--nostyle mui--text-light" @click="closeForm($event)">Cancel</button>
               </div>
               <div v-if="activeMember" class="modal-form__action-box--revoke mui--text-center">
-                <button @click="fetchForm($event, deleteURL)" title="Delete" class="mui-btn mui-btn--nostyle mui--text-subhead"><span class="mui--text-light">Revoke this membership - </span><span class="mui--text-bold">Revoke</span></button>
+                <button @click="fetchForm($event, deleteURL)" class="mui-btn mui-btn--nostyle mui--text-subhead"><span class="mui--text-light">Remove this member - </span><span class="mui--text-bold">Remove</span></button>
```
Comment (jace) [inline funnel/templates/project_membership.html.jinja2:97]: Same here.

File: funnel/views/membership.py (hunk 1 above)
Comment (jace) [inline funnel/views/membership.py:192]: "The member’s roles have been updated"

File: funnel/views/membership.py [hunk 7]
```diff
@@ -493,6 +495,7 @@ def edit(self):
...
```
Comment (jace) [inline funnel/views/membership.py:498]: "The member’s roles have been updated"

File: funnel/assets/js/membership.js [hunk 8]
```diff
@@ -24,6 +24,9 @@ const Membership = {
           if (member.is_usher) count += 1;
           return count - 1;
         },
+        getInitials(name) {
+          return Utils.getInitials(name);
+        },
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/assets/js/membership.js:29]: This is to call global util function from vue template. Let me check a better way to do it.

File: funnel/views/membership.py [hunk 9]
```diff
@@ -249,6 +250,7 @@ def delete(self):
                     )
                 return {
                     'status': 'ok',
+                    'message': _("The member have been removed"),
```
Comment (jace) [inline funnel/views/membership.py:253]: `has`

File: funnel/views/membership.py (hunk 7 above)
Comment (jace) [inline funnel/views/membership.py:498]: In this case, `have`

File: funnel/assets/js/util.js [hunk 10]
```diff
@@ -223,6 +223,21 @@ export const Utils = {
       $('input[name="form_nonce"]').val(response.form_nonce);
     }
   },
+  getInitials(name) {
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/assets/js/util.js]: Moved to baseframe https://github.com/hasgeek/baseframe/pull/300

**Pull request merged**